src/KeyringController.test.ts

```diff
@@ -1075,6 +1121,19 @@ describe('KeyringController', () => {
         'someapp.origin.io',
       ]);
     });
+
+    it('should throw error if keyring for account does not support getAppKeyAddress', async () => {
```

Comment:
Nit: It looks like we aren't using "should" in the names of these tests.

So, for example:

``` suggestion
    it('throws error if keyring for account does not support getAppKeyAddress', async () => {
```

but a similar suggestion would apply to the other tests in this file.

Reply:
Good point, the pattern is not clear in the file: some tests use "should", while others use "throws, forwards, adds, etc.."

I changed all the occurrences that I found, removing "should" in 988974e